domain/domain_test.go

```diff
@@ -133,19 +135,91 @@ func interestingGoroutines() (gs []string) {
 	return
 }
 
```

Comment:
Could we add a "TODO" here?

Reply:
// For debug only, will be removed later. In L95, 
I think it's enough to remind us.

Reply:
Got it.